src/capi/azext_capi/custom.py

```diff
@@ -6,6 +6,7 @@
 """This module implements the behavior of `az capi` commands."""
 
 # pylint: disable=missing-docstring
+# pylint: disable=too-many-lines
```

Comment:
maybe it's time to break out this file into several smaller files?

Reply:
I agree, I have opened an issue #88